.eslintrc.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  extends: [
```

Comment:
do we need to keep other extends in line 17?

Reply:
Removed obsolete extends